src/sentry/search/utils.py

```diff
@@ -145,6 +147,26 @@ def get_date_params(value, from_field, to_field):
         })
     return result
 
+reserved_tag_names = frozenset([
```

Comment:
I mostly agree but i'm inclined to keep these like this right now, because I'm slightly anxious about these becoming outdated or otherwise breaking.